doc/nestml_language/nestml_language_concepts.rst

```diff
@@ -339,15 +340,17 @@ If a comment shall be attached to an element, no white lines are allowed.
 
 .. code-block:: nestml
 
-   V_m mV = -55 mV # I am a comment of the membrane potential
+   # I am not a comment of the membrane potential.
+
    # I am a comment of the membrane potential.
```

Comment:
```suggestion
```

Reply:
Why do you suggest to remove this?

Reply:
to illustrate that if a whiteline separates a pre comment and its correponding element, it is not picked up as belonging to that element (this is the way it is, right?)

Reply:
Yes, you are right. That is why I have a pre-comment attached to that element (on L345) and one more comment before that separated by whitespace (on L343). I think your suggestion points to removing the pre-comment on L345. Would you still like for me to remove that?